### ISSUE TITLE
No output VCF files

### ISSUE THREAD

**Issue by Ballote** [freeseek/gtc2vcf#36]
Hello

I see it's necessary two steps to convert from .CEL to .VCF. In the first step is generated xxxxx.AxiomGT1.chp files (where xxxxx is the name of the original file) is this correct?

Now, I'm having problem with the second step. When I run that part of the program I have no errors but also I can't find the VCF files. This is the code I'm running:

bcftools +affy2vcf \
  --no-version -Ou \
  --csv "GenomeWideSNP_6.na35.annot.csv" \
  --fasta-ref "human_g1k_v37.fasta" \
  --chps /home/adrianib/Proyecto/cc-chp \
  --snp /home/adrianib/Proyecto/AxiomGT1.snp-posteriors.txt \
  --extra result.tsv | \
  bcftools sort -Ou -T ./bcftools-sort.XXXXXX | \
  bcftools norm --no-version -Ob -o result.bcf -c x -f "human_g1k_v37.fasta" && \
  bcftools index -f result.bcf

I see there is no command to indicate the output folder as in the first step. This could be the reason I don't have output VCF files?

In summary, I have this:
Original file: xxxxx.CEL
1st step (CEL to CHP): xxxxx.AxiomGT1.chp
2nd step (CHP to VCF): ?

And my question is: Should I have a xxxxx.VCF file at the end of the second step?

Thanks for your help
Adrian

**Comment by freeseek** [comment]
Yes, the idea is that you first use the Affymetrix Power Tools to convert the CEL files to CHP files, and then +affy2vcf to convert the CHP files to VCF files. The code you showed seems right (make sure the `\` escapes are used if you have the command over multiple rows). It is difficult for me to advise as to why you did not get a VCF file in the end without you sharing the command output logs.

**Comment by freeseek** [comment]
It looks fine. Are you sure you don't have a `result.bcf` file in the end?

**Comment by Ballote** [comment]
Yes, I have 3 output files: result.bcf, result.bcf.csi and result.tsv. 

But shouldn't it have a .VCF file for every .CHP file? or are the VCF files inside result.BCF? I do not understand

**Comment by freeseek** [comment]
The `bcftools sort` command generates one VCF file (in this case a binary VCF file, since you used the `-Ob` option). The `+affy2vcf` tool converts a set of CHP files into one single VCF. If you wanted one VCF file per CHP file, you would have had to run the command once for each conversion and input one CHP file at a time.

**Comment by freeseek** [comment]
You are welcome. :-)
